website/src/components/Dashboard/LeaderboardTable.tsx

```diff
@@ -1,32 +1,26 @@
-import { Box, Link, Text, useColorModeValue } from "@chakra-ui/react";
+import { Card, CardBody, Flex, Link, Text } from "@chakra-ui/react";
```

Comment:
I'd suggest dropping the changes to this file.  I think it's changed a lot and has the relevant changes you're working on already.

Reply:
Is that in a different PR? The changes build off the current version and this just swaps native elements to Chakra components

Reply:
Yeah that just went in with some changes to various table components.